README.md

```diff
@@ -33,7 +33,9 @@
 - StackViewHelpers
 - TableViewHelpers
 - TintedButton
-- Views (GradientView, TextView)
+- Views:
+ - [GradientView](Pod/Classes/Views/Gradient/README.md) - UIView containing a color gradient.
+ - [TextView](Pod/Classes/Views/Textview/README.md) - UITextView that can present placeholder text or expand it’s height to accommodate text content.
```

Comment:
name mismatch with the new name of this class

Reply:
Not sure what you mean. The `TextView` folder contains more than one class derived from UITextView. I guess we could make "UITextView" plural here for more clarity.

Reply:
Ohhh got it. I thought this was matching the name of the class. Maybe `Text Views`?